homeassistant/components/mqtt/mixins.py

```diff
@@ -586,6 +588,11 @@ def _setup_from_config(self, config):
     async def _subscribe_topics(self):
         """(Re)Subscribe to topics."""
 
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if the entity should be enabled when first added to the entity registry."""
+        return self._config.get(CONF_ENABLED_BY_DEFAULT)
```

Comment:
Since the key is always there, use square brackets. If you want to have teh default live here and not in the config schema, remove `default=True` from `MQTT_ENTITY_COMMON_SCHEMA`